### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by nathandao**
Follow the README template from https://github.com/gitpoint/git-point#readme

The Ylitse introduction was taken from Spice Program website :D 

Do feel free to make comments / adjust content as you see fit!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+- `npm run ios` to start the packager and run the app in the iOS simulator
+- `npm run android` to start the packager and run the app in the the Android device/emulator
+
+## Sponsors
+
+Ylitse app is proudly sponsored by [SOS Lapsikylä](https://www.sos-lapsikyla.fi/).
+
+## Acknowledgments
+
+TODO: update content.
```
Comment (nathandao) [inline README.md]: There's at least Azure, but we are not using it yet, so probably not adding here for now.

Besides that, what else would make sense? A list of major os libs that we are using? (react, react-native, etc ...)?

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
...
+
+<p align="center">
+  <a href="#">
+    <img alt="Download on the App Store" title="App Store" src="http://i.imgur.com/0n2zqHD.png" width="140">
+  </a>
+
+  <a href="#">
+    <img alt="Get it on Google Play" title="Google Play" src="http://i.imgur.com/mtGRPuM.png" width="140">
+  </a>
+</p>
```
Comment (pheis) [inline README.md]: wow some html stuff and images!

File: README.md (hunk 1 above)
Comment (pheis) [reply README.md]: https://gitlab.com/ylitse/ylitse-mobile Ivan had written some libs on old README.

Comment (pheis) [reply README.md]: But not sure at all if it makes sense or not.

Comment (pheis) [reply README.md]: At some point it would make sense to talk about ylitse-api but maybe it could be added later?

File: README.md (hunk 2 above)
Comment (nathandao) [reply README.md]: yeapp : D Only some xml in your markdown ¯\_( ͡° ͜ʖ ͡°)_/¯

**Pull request closed without merging**